CMakeLists.txt

```diff
@@ -1,16 +1,22 @@
+# Setup CMake project
 cmake_minimum_required(VERSION 3.11)
 project(HELLO CXX)
 
+# Include module library
 include(modules.cmake)
 
+# Display debug info
 modules_get_latest_cxx_std(std_latest_ver)
-message("std_latest_ver=${std_latest_ver}")
-
 modules_supported(cxx_modules)
-message("cxx_modules=${cxx_modules}")
+message("C++ compiler version   = ${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION}")
```

Comment:
I don't think this is needed because CMake prints it already, e.g.

```
-- The CXX compiler identification is AppleClang 14.0.3.14030022
```

Reply:
removed